dataflow/src/org/checkerframework/dataflow/cfg/CFGVisualizer.java

```diff
@@ -47,10 +53,30 @@
      *            indicate that this information should not be output.
      * @return Possible analysis results, e.g. generated files.
      */
-    <A extends AbstractValue<A>, S extends Store<S>, T extends TransferFunction<A, S>>
     /*@Nullable*/ Map<String, Object> visualize(ControlFlowGraph cfg, Block entry,
             /*@Nullable*/ Analysis<A, S, T> analysis);
 
+    // TODO: javadoc
+    void visualizeStore(S store);
+
+    void visualizeStoreHeader(String classCanonicalName);
+
+    void visualizeStoreFooter();
+
+    void visualizeLocalVariable(FlowExpressions.LocalVariable localVar, A value);
+
+	void visualizeThisValue(A value);
```

Comment:
Why are these methods indented differently? If there is some grouping, express it in comments.